packages/quick_actions/quick_actions_platform_interface/lib/types/shortcut_item.dart

```diff
@@ -20,6 +21,11 @@ class ShortcutItem {
   /// Localized title of the item.
   final String localizedTitle;
 
+  /// Localized subtitle of the item.
+  ///
+  /// This is ignored on Android, since it's not supported.
```

Comment:
Platform interface packages should not make claims about what implementation packages, which are a higher level, do or don't do. This should just say that it's ignored by platforms that don't support it.